execution/execution.go

```diff
@@ -204,6 +217,10 @@ func newOperator(expr parser.Expr, storage *engstore.SelectorPool, opts *query.O
 		}
 
 	case *parser.StepInvariantExpr:
+		switch t := e.Expr.(type) {
```

Comment:
What case does this handle?

Reply:
This is fixing a regression. The case was added here but got removed at one point: https://github.com/thanos-community/promql-engine/pull/75/files#diff-9fd654a57fa6fe30931b10efaf6f7d1eaf5bebdac900785e7d3b79faa9bfcc85R166

Reply:
Just tested that this fixes a problem for us that I've mentioned here: https://github.com/thanos-community/promql-engine/pull/101#issuecomment-1293290420